CruzHacks/cruzhacks-2021-website
# adding buttons to website body

Problem
=======
As a user,
I want to use a navigation bar
to navigate between different useful pages/links.

Solution
========
I broke the navigation bar into two sections, which are separated by `space-between`. One section includes just the title, while the other section includes links to the sponsor and hacker application. When the screen size decreases, the button container wraps beneath the title for adequate responsiveness. Each button links to the corresponding hacker and sponsor application, which are passed in as props and is defined in its parent container. 

Change summary:
---------------
* Finished first draft of navigation bar 2ba3a0a

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/navbar`
2. Start local development server using `yarn start`
3. View the navigation bar and verify that the component is responsive and that the buttons link to the correct forms.
